tests/plugins/query_insights/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+
+services:
+  opensearch-cluster:
+    image: ${OPENSEARCH_DOCKER_HUB_PROJECT:-opensearchproject}/opensearch:${OPENSEARCH_VERSION:-latest}${OPENSEARCH_DOCKER_REF}
+    ports:
+      - 9200:9200
+      - 9600:9600
+    environment:
+      - OPENSEARCH_INITIAL_ADMIN_PASSWORD=${OPENSEARCH_PASSWORD:-myStrongPassword123!}
+      - OPENSEARCH_JAVA_OPTS=${OPENSEARCH_JAVA_OPTS}
+      - discovery.type=single-node
+      - search.insights.top_queries.latency.enabled=true
+      - search.insights.top_queries.cpu.enabled=true
+      - search.insights.top_queries.memory.enabled=true
```

Comment:
Are these custom parameters required, meaning is query insights not working in the default configuration? If that's the case then this is right, otherwise add tests to `default`.

Reply:
added tests for default

Reply:
I meant removing the `tests/plugins/query_insights` folder and moving the tests to the `tests/default` folder.

Reply:
query insights features are all in a plugin that needs to be installed first. In this case shouldn't we write the tests in the plugin folder?

Reply:
Cool. Add this to the CI/CD matrix in https://github.com/opensearch-project/opensearch-api-specification/blob/main/.github/workflows/test-spec.yml#L25 and it should be it.

Reply:
And since everything is called "insights", should the folder be called "tests/plugins/insights" too? Or is the plugin actually called query_insights?

Reply:
Yes. The plugin is actually called `query_insights`. I added it to the CI/CD matrix as well.

Reply:
Cool. Iterate to green.